resource_packs/packs.json

```diff
@@ -2274,6 +2288,11 @@
 			"id": "parity",
 			"name": "Parity",
 			"packs": [
+				{
+					"id": "java_menu_title",
+					"name": "Java Menu Title",
+					"description": "Gives the title on the main menu screen its look from Minecraft: Java Edition"
+				},
```

Comment:
This needs to be removed as this pack was removed in this pr #220

Reply:
already removed, you are looking at old commits.